src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -806,10 +806,21 @@ namespace ReportSizingManager {
 					} else if ( ZoneEqUnitHeater || ZoneEqVentedSlab ) {  // for unit ventilator the cp value is calculated at 5.05(InitConvTemp) for the child and 60.0C for the unit ventilator //|| ZoneEqUnitVent
 						AutosizeDes = ZoneEqSizing( CurZoneEqNum ).MaxHWVolFlow;
 					} else {
-						CoilInTemp = FinalZoneSizing( CurZoneEqNum ).DesHeatCoilInTemp;
```

Comment:
These new sections get the parent air flow rate and then calculate the OA fraction so a "revised" variable can be calculated based on the zone temp/humrat and OA temp/humrat.